### PR TITLE
update readme.adoc

### PR DESCRIPTION
**Pull request by anuragknoldus**
I  just want to add this change because when we place jar into $Neo4j_Home/plugins than it is very useful to find the count of procedure into any specific package. So we can know that we have latest apoc-procedure or not.

Thanks.

### CODE REVIEW

File: readme.adoc [hunk 1]
```diff
@@ -81,6 +81,16 @@ RETURN name AS undocumented
 
 // end::help[]
 
+=== Builtin Pakage and Procedure count
+
+// tag::procedurecount[]
+
+To find the procedure count with the pakage in database:
```
Comment (jexp) [inline readme.adoc]: typo in package

Comment (jexp) [inline readme.adoc]: also add the Cypher statement in a [source,cypher] block not just the image

File: readme.adoc [hunk 2]
```diff
@@ -81,6 +81,16 @@ RETURN name AS undocumented
 
 // end::help[]
 
+=== Builtin Pakage and Procedure count
```
Comment (jexp) [inline readme.adoc]: typo in Pa_c_kage

**Comment by anuragknoldus** [comment]
Sorry Michael for the typing error.
I will update it and than again send the pull request.

**Pull request closed without merging**